modules/node_modules/@colony/purser-trezor/staticMethods.js

```diff
@@ -70,89 +70,88 @@ export const signTransaction = async ({
+       * Eg: '3' to be '03', `12c` to be `012c`
+       */
+      nonce: hexSequenceNormalizer(
         /*
-         * The transaction object needs to be seeded with the (R) and (S) signature components with
-         * empty data, and the Reco(V)ery param as the chain id (all, im hex string format).
+         * @TODO Add `bigNumber` `toHexString` wrapper method
          *
-         * See this issue for context:
-         * https://github.com/LedgerHQ/ledgerjs/issues/43
+         * Flow confuses bigNumber's `toString` with the String object
+         * prototype `toString` method
          */
-        r: hexSequenceNormalizer(
-          multipleOfTwoHexValueNormalizer(String(SIGNATURE.R)),
-        ),
-        s: hexSequenceNormalizer(
-          multipleOfTwoHexValueNormalizer(String(SIGNATURE.S)),
-        ),
-        v: hexSequenceNormalizer(
-          /*
-           * @TODO Add `bigNumber` `toHexString` wrapper method
-           *
-           * Flow confuses bigNumber's `toString` with the String object
-           * prototype `toString` method
-           */
-          /* $FlowFixMe */
-          multipleOfTwoHexValueNormalizer(chainId.toString(16)),
-        ),
-      },
+        /* $FlowFixMe */
+        multipleOfTwoHexValueNormalizer(nonce.toString(16)),
+      ),
+      value: hexSequenceNormalizer(
+        /*
+         * @TODO Add `bigNumber` `toHexString` wrapper method
+         *
+         * Flow confuses bigNumber's `toString` with the String object
+         * prototype `toString` method
+         */
+        /* $FlowFixMe */
+        multipleOfTwoHexValueNormalizer(value.toString(16)),
+      ),
+      data: hexSequenceNormalizer(inputData),
+      /*
+       * The transaction object needs to be seeded with the (R) and (S) signature components with
+       * empty data, and the Reco(V)ery param as the chain id (all, im hex string format).
+       *
+       * See this issue for context:
+       * https://github.com/LedgerHQ/ledgerjs/issues/43
+       */
+      r: hexSequenceNormalizer(
+        multipleOfTwoHexValueNormalizer(String(SIGNATURE.R)),
+      ),
+      s: hexSequenceNormalizer(
+        multipleOfTwoHexValueNormalizer(String(SIGNATURE.S)),
+      ),
+      v: hexSequenceNormalizer(
+        /*
+         * @TODO Add `bigNumber` `toHexString` wrapper method
+         *
+         * Flow confuses bigNumber's `toString` with the String object
+         * prototype `toString` method
+         */
+        /* $FlowFixMe */
+        multipleOfTwoHexValueNormalizer(chainId.toString(16)),
+      ),
+    },
+    {
+      chain: chainId,
       /*
-       * Only set (and normalize) the destingation address if one was
+       * Only send (and normalize) the destination address if one was
        * provided in the initial transaction object.
        */
-      typeof to !== 'undefined' ? { to: addressNormalizer(to) } : {},
-    ),
+      ...(to ? { to: addressNormalizer(to) } : {}),
```

Comment:
I'll need to manually check the Trezor service, since I remember it going crazy if we passed in values it didn't expect.